xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/DXVAHD.cpp

```diff
@@ -848,6 +866,72 @@ bool CProcessorHD::IsBT2020Supported()
   return false;
 }
 
+bool CProcessorHD::IsHDR10PassthroughSupported(const DXGI_FORMAT dxgi_format)
```

Comment:
Do you mind renaming to IsPQ010PassthroughSupported? PQ010 is HDR10 without metadata, which don't matter here